commands-yml/commands/session/timeouts/async-script.yml

```diff
@@ -41,7 +41,7 @@ driver_support:
     xcuitest: false
     uiautomation: false
   android:
-    uiautomator2: false
+    espresso: false
```

Comment:
as same as above (uiautomator2)

Reply:
@KazuCocoa good catch, it must have been replaced by accident